main.go

```diff
@@ -1,9 +1,11 @@
 package main
 
-import "github.com/vlpolak/swtgo/server"
+import (
+	"github.com/vlpolak/swtgo/ws"
+)
 
 func main() {
-	//ws.StartServer()
-	s := server.CreateServer()
-	s.Serve()
+	ws.StartServer()
+	//s := server.CreateServer()
+	//s.Serve()
```

Comment:
It seems, the app is not able to handle any requests except `http://localhost:8080/' after that change.